datalad/support/gitrepo.py

```diff
@@ -59,7 +134,7 @@ def __init__(self, path, url=None):
                 lgr.error(str(e))
                 raise
 
-        if not exists(join(path, '.git')):
+        if not exists(p_join(path, '.git')):
```

Comment:
I thought I left those comments somewhere -- either import as is (join) or opj  as we used elsewhere... no p_joins